charts/ratify/values.yaml

```diff
@@ -71,12 +73,57 @@ oras:
     awsEcrBasicEnabled: false
 
 provider:
-  auth: ""                  # tls, mtls
+  auth: "tls"                  # auth must be tls or mtls
   tls:
-    skipVerify: true        # skip TLS verification
-    crt: ""                 # crt used by ratify (httpserver)
-    key: ""                 # key used by ratify (httpserver)
-    cabundle: ""            # base64 encoded CA bundle used for the 'caBundle' property for the ratify provider within gatekeeper
+    crt: |  # crt used by ratify (httpserver), please provide your own crt
+      -----BEGIN CERTIFICATE-----
+      MIIC+TCCAeGgAwIBAgIUYa4y/XTsyvfUKSeK0g4YpcnZw0wwDQYJKoZIhvcNAQEL
+      BQAwNTEPMA0GA1UECgwGTXkgT3JnMSIwIAYDVQQDDBlFeHRlcm5hbCBEYXRhIFBy
+      b3ZpZGVyIENBMB4XDTIzMDEwOTA2MzMyM1oXDTI0MDEwOTA2MzMyM1owGTEXMBUG
+      A1UEAwwOcmF0aWZ5LmRlZmF1bHQwggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEK
+      AoIBAQDoyPCu2MI8X8j+4PygKABAIFf6y3w4Gx0uyumZvGHljAloZYU7sx6XtiPB
+      zxZ8CJz/Zd3Bb/3qG2gK6JBeoTZmZwdVjrN83Vj7WiJwZuPvxtGDLZm/woLhQQBK
+      smFAO60KQYqn7W3M3/wihwhLOBPFlphH7Ewe1MzKXh9PqAKnPmwHUOZ5p7noxo/c
+      LDUbZ/RBAkM915VoqdzQ0mVLVwtMGGCgpwpW/dr1aGpLSmrem1H5GNdDNijvKq2V
+      JxtoR+TYfdCrVfPxsvQZrqQ9xGhPOa5jwSM7nDmQmsn4L4Zea7mis+FdCZ2+sZ3U
+      gGyzWZI+WPA98z7rUewIU6KpbmGTAgMBAAGjHTAbMBkGA1UdEQQSMBCCDnJhdGlm
+      eS5kZWZhdWx0MA0GCSqGSIb3DQEBCwUAA4IBAQArrVshmau1ZcfA09zuTG2l8ijn
+      2iQizg8T5NEUVQM8k/xCF5nspXhdQ5v82uDO2p81impJ8R9pwJMnl91N/At8zc+U
+      jHT/CJ/DevExAlT5IBL7P8y0eMGaKKEH0b8DpK3/HR7DBEosCzKXNpQFmAue1NDr
+      oUhwc1NTJ+srwY5CBP2aH/uTwJqPPFG4h+sVPZ1Fu6EbMv/1Q+rmFWfCi0C1gHXn
+      6MQCdTHQAVHSAwrpK3hc7/LIgXG35bgBkS+jAQSkikYrwNnv/O0FmL9IFI0iv+XO
+      l9xnBvSWbzl+ZF/wm6F8jNPmn4coMRzjsizOfxuNjdZML9UiQQpLY0TXk3jn
+      -----END CERTIFICATE-----
+    key: |  # key used by ratify (httpserver), please provide your own key
+      -----BEGIN PRIVATE KEY-----
```

Comment:
Question to reviewers: should we add the private key for quick-start setup? Or we can leave it empty and update the quick-start steps to let users generate it.